### PR TITLE
[KAA-1090] Prevent thread blocking by PingTask with dispatch_after fu…

### PR DESCRIPTION
**Pull request by am1kk**
…nction.

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

File: client/client-multi/client-objective-c/Kaa/channel/impl/channels/DefaultOperationTcpChannel.m [hunk 1]
```diff
@@ -650,26 +650,28 @@ - (instancetype)initWithChannel:(DefaultOperationTcpChannel *)channel {
...
-    
-    @try {
-        DDLogInfo(@"%@ Executing ping task for channel [%@]", TAG, [self.channel getId]);
-        [self.channel sendPingRequest];
-        if (self.isCancelled) {
-            DDLogInfo(@"%@ Can't schedule new ping task for channel [%@]. Task was cancelled.", TAG, [self.channel getId]);
-        } else {
-            [self.channel schedulePingTask];
+    dispatch_time_t time =  dispatch_time(DISPATCH_TIME_NOW, (int64_t)PING_TIMEOUT_SEC * NSEC_PER_SEC);
+    dispatch_queue_t queue =  dispatch_queue_create("com.kaaproject.pingtask", NULL);
```
Comment (sergiyk0) [inline client/client-multi/client-objective-c/Kaa/channel/impl/channels/DefaultOperationTcpChannel.m]: Queue ID should be "org.kaaproject.pingtask"

Comment (abohomol) [inline client/client-multi/client-objective-c/Kaa/channel/impl/channels/DefaultOperationTcpChannel.m:654]: Creating separate queues for each can lead to large memory issues.
However starting separate NSOperation just to schedule another block in another queue isn't good idea in general.

Comment (sergiyk0) [inline client/client-multi/client-objective-c/Kaa/channel/impl/channels/DefaultOperationTcpChannel.m:654]: Better to take some global queue with low priority(QOS_CLASS_UTILITY probably fits best) than create it manually every time.

**Pull request closed without merging**